Scuti/Resources/ScutiGamepad.inputactions

```diff
@@ -0,0 +1,876 @@
+{
```

Comment:
@Jpablogs8 This file is going to be included in all games that integrate Scuti. Should it be in their projects? If so, we need to give it a better name than testscutigamepad. If not, please remove it.

Reply:
Ok!